goas/cl/ledger/src/bundle.rs

```diff
@@ -0,0 +1,57 @@
+use crate::error::Result;
+
+pub struct ProvedBundle {
+    pub bundle: cl::Bundle,
+    pub risc0_receipt: risc0_zkvm::Receipt,
+}
+
+impl ProvedBundle {
+    pub fn prove(bundle: &cl::Bundle, balance_witness: &cl::BalanceWitness) -> Self {
```

Comment:
nit: should we call this bundle witness just for naming consistency?

Reply:
in this case, we are actually not taking the bundle witness here. This is the bundle that lands on chain.

The proof is simply saying that if you sum up all the balance commitments in the bundle, it equals `Balance::zero(balance_witness)`.

I.e. it's a proof that the bundle balance is a commitment to zero.

Reply:
yesyes, just talking about names, we have this https://github.com/logos-co/nomos-pocs/blob/e518bf363cf7898555e9583ddac9f0aaf017a5a3/goas/cl/cl/src/bundle.rs#L17 which looks like it could be used here?

Reply:
ah, yeah, I could use that here

Reply:
done